src/pocketmine/Player.php

```diff
@@ -4135,6 +4134,8 @@ public function addWindow(Inventory $inventory, int $forceId = null, bool $isPer
 	public function removeWindow(Inventory $inventory, bool $force = false){
 		$id = $this->windows[$hash = spl_object_hash($inventory)] ?? null;
 
+		(new InventoryCloseEvent($inventory, $this))->call();
```

Comment:
This is going to fire when using removeWindow() with an inventory that wasn't opened to begin with.